### PR TITLE
138: Add actuals and goals totals

### LINKED ISSUE #138: Add actuals and flight-status to the dashboard
Where the Goal column exists now:

Also add 2 more columns:
- Actuals (_NOT SORTABLE_)
  - it's the flight `actualCount`
- Flight Status (whether the last allocation-engine cron succeeded or failed.  In other words ... do we have a _plan_ to meet the totalGoal or not).
  - it's the flight `status` and/or `statusMessage`.  The status will be `"ok"` or `"error"`, and the `statusMessage` blank/null or a fairly long and descriptive string.  May be shortest/best to just capitalize `status` that and throw it in the column.
  - TODO: @cavis needs to make this sortable (1-liner in augury)

And then using simple mathematic equations of summation [ie addition] provide the total for actuals / totalGoals in a new table header row. (not in the footer.)

### PR DESCRIPTION
**Pull request by rpeterman-gp**
Closes #138 

- adds Actuals column to flights table
- adds F. Status column to flight table

### To Review
- [ ] Checkout branch.
- [ ] Run `yarn`.
- [ ] Run 'docker-compose build`.
- [ ] Run `docker-compose up`.
- [ ] Run `docker start http-proxy`.
- [ ] Go to [grove.prx.docker](https://grove.prx.docker) and login or masquerade.
- [ ] Ensure flights table now has an _Actuals_ column before the _Goal_ column.
- [ ] Ensure a footer row in shown with _Totals_ of the actuals and goals for the current page.
- [ ] Ensure flights table has a _F. Status_ column showing the current status of the flight.

### CODE REVIEW

File: src/app/dashboard/flight-table/flight-table.component.html [hunk 1]
```diff
@@ -106,5 +139,5 @@
     [pageIndex]="routedParams?.page - 1 || 0"
     [pageSizeOptions]="pageSizeOptions"
     showFirstLastButtons
-  ></mat-paginator>
```
Comment (sandikbarr) [inline src/app/dashboard/flight-table/flight-table.component.html]: Whoops, this extra bit got left in there

**Review by sandikbarr — CHANGES_REQUESTED** [review]
just the extra `Hello` bit in the flight-table.component.html otherwise looks good

File: src/app/dashboard/flight-table/flight-table.component.html (hunk 1 above)
Comment (rpeterman-gp) [reply src/app/dashboard/flight-table/flight-table.component.html]: Fixed

**Review by sandikbarr — APPROVED** [review]
:) looks good

**Pull request merged**